lib/devtools/config.rb

```diff
@@ -150,7 +150,7 @@ class Mutant < self
     # Devtools configuration
     class Devtools < self
       FILE = 'devtools.yml'.freeze
-      DEFAULT_UNIT_TEST_TIMEOUT = 0.1  # 100ms
+      DEFAULT_UNIT_TEST_TIMEOUT = 0.1 # 100ms
```

Comment:
> Is there a rubocop setting that might allow 2 spaces for trailing comments?

@backus do you mind to look that up? and adjust the setting if available, or deactivating the cop otherwise?